Automattic/jetpack
# AI Featured Image: Add modal and content

Getting the modal from `Proofread` and make it shared, reuse it in `Featured Image` and implement an initial content for the modal.

## Proposed changes:

* Create `AiAssistantModal` component.
* Reuse it in `Proofread` and `Featured Image`.
* Add content for modal with fake generation in `Featured Image`.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Make sure you have `beta` blocks enabled: `define( 'JETPACK_BLOCKS_VARIATION', 'beta' )`.
* Open `Jetpack Plugin Sidebar` and click in the `AI Assistant` panel.
* You should see `Generate Image` button enabled.
* Click on it, it will show an loading and then an image with two buttons.
* Make sure the `Generate feedback` is working as expected too.

#### Demo 💻 

https://github.com/Automattic/jetpack/assets/1663717/37e43073-31eb-46f8-a218-5772acc0d1b1
